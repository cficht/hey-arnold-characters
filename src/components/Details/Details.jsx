import React from 'react';
import PropTypes from 'prop-types';
// import { Link } from 'react-router-dom';

const Details = ({ name, image }) => (
  <>
    <p>{name}</p>
    <img src={image} />
  </>
);

Details.propTypes = {
  name: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
};

export default Details;

