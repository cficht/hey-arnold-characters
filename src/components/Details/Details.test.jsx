import React from 'react';
import { shallow } from 'enzyme';
import Details from './Details';

describe('Details component', () => {
  it('renders Details', () => {
    const wrapper = shallow(<Details 
      name="Arnold Shortman" 
      image="https://vignette.wikia.nocookie.net/heyarnold/images/f/f5/Arnold.png/revision/latest?cb=20181027162333"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
