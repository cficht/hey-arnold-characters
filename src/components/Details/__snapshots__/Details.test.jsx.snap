// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details component renders Details 1`] = `
<Fragment>
  <p>
    Arnold Shortman
  </p>
  <img
    src="https://vignette.wikia.nocookie.net/heyarnold/images/f/f5/Arnold.png/revision/latest?cb=20181027162333"
  />
</Fragment>
`;
